dbt/pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[project]
+name = "dbt"
+version = "0.0.1"
+readme = "README.md"
+requires-python = ">=3.11"
+dependencies = [
+  "dbt-athena-community==1.8.2",
+  "dbt-core==1.8.3",
+]
+
+[project.optional-dependencies]
+dbt_tests = [
+  "PyAthena~=3.8.3",
+  "lxml~=5.1.0",
+  "numpy==1.26.4",
+  "openpyxl~=3.1.5",
+  "pandas~=2.2.2",
+  "pyarrow==15.0.2",
+  "pyathena~=3.8.3",
+  "simplejson~=3.19.2",
+]
+ci_checks = [
+  "pyyaml==6.0.2",
+]
+
+[tool.setuptools]
+packages = []
```

Comment:
This little addendum is required or pip will try to install all your subdirectories as modules/packages.